Cargo.toml

```diff
@@ -48,21 +47,33 @@ failure = "0.1"
 x509-parser = "0.6.0"
 sspi = "0.3.0"
 ironrdp = { version = "0.4.1", git = "https://github.com/Devolutions/IronRDP", rev = "ca53209c84c55b41" }
-hyper = "0.12.32"
-tungstenite = "0.10"
+hyper = "0.13"
 sha1 = "0.6.0"
 base64 = "0.9.1"
 slog-stdlog = "4.0"
 ring = "0.16.0"
 spsc-bip-buffer = { git = "https://github.com/Devolutions/spsc-bip-buffer.git", branch = "master" }
 indexmap = "1.0"
 
+[dependencies.saphir]
```

Comment:
TODO: change to mainstream repo / published version when PR https://github.com/richerarc/saphir/pull/128 will be merged

Reply:
Looks like it just got merged :tada: